tekdoc/main.go

```diff
@@ -17,15 +17,72 @@ limitations under the License.
 package main
 
 import (
+	"flag"
 	"fmt"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	"github.com/tektoncd/pipeline/pkg/client/clientset/versioned/scheme"
+	"io/ioutil"
+	"log"
+	"os"
+	"text/template"
+)
 
-	_ "github.com/tektoncd/pipeline/pkg/pod"
+var (
+	filename = flag.String("f", "", "Name of the file to parse")
 )
+func check(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
+type taskStruct struct {
```

Comment:
Instead of using a new struct, could we just use the already defined task struct? We should be able to reference the fields directly like we do with this type!

Reply:
I'll work on it and get back

Reply:
> Instead of using a new struct, could we just use the already defined task struct? We should be able to reference the fields directly like we do with this type!

@wlynch Could you please help me finding the existing struct which I can reuse in my code?